apps/artemis-web/components/status-table/status-table.tsx

```diff
@@ -134,18 +139,30 @@ class StatusTable extends Component<StatsType, any> {
           <tbody>
             {STATS_DATA && backend ? (
               backend.map((process, i) => {
-                return (
-                  <ModuleState
-                    key={i}
-                    process={process}
-                    modules={modulesSet}
-                    index={i}
-                    tooltip={
-                      tooltips[process.name.slice(0, process.name.indexOf('-'))]
-                    }
-                    date={this.state.date}
-                  ></ModuleState>
-                );
+                if (
+                  !backList.includes(
+                    process.name.slice(0, process.name.indexOf('-'))
+                  )
+                ) {
+                  backList.push(
+                    process.name.slice(0, process.name.indexOf('-'))
+                  );
+
+                  return (
+                    <ModuleState
+                      key={i}
```

Comment:
please put some comments in the code, it is very hard for an outside dev to understand why e.g., you are using this index etc. Briefly comment your blocks of code, otherwise we will not be able to maintain them on the long run.